app/routes/application.js

```diff
@@ -1,14 +1,14 @@
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
 
-export default Route.extend({
-  navbar: service(),
-  metrics: service(),
-  fastboot: service(),
-  router: service(),
+export default class AplicationRoute extends Route {
+  @service navbar;
+  @service metrics;
+  @service fastboot;
+  @service router;
```

Comment:
For posterity, here is the documentation for how to report route transitions to `ember-metrics`:

https://github.com/adopted-ember-addons/ember-metrics#native-class-usage